api/src/services/submission-service.ts

```diff
@@ -162,10 +163,20 @@ export class SubmissionService extends DBService {
    * @return {string}
    * @memberof SubmissionService
    */
-  async getSubmissionRecordSONByDatasetId(datasetId: string): Promise<string> {
+  async getSubmissionRecordJSONByDatasetId(datasetId: string): Promise<string> {
     return this.submissionRepository.getSubmissionRecordJSONByDatasetId(datasetId);
   }
 
+  /**
+   *
+   *
+   * @return {*}  {Promise<string>}
+   * @memberof SubmissionService
+   */
+  async getDatasetIdsFromDB(): Promise<IDatasetId[]> {
```

Comment:
If we are no longer using this, lets get rid of it (and whatever other related code)